ArduPlane/ArduPlane.cpp

```diff
@@ -194,9 +194,9 @@ void Plane::ahrs_update()
 void Plane::update_speed_height(void)
 {
     if (auto_throttle_mode) {
-	    // Call TECS 50Hz update. Note that we call this regardless of
-	    // throttle suppressed, as this needs to be running for
-	    // takeoff detection
+        // Call TECS 50Hz update. Note that we call this regardless of
```

Comment:
We'd prefer it if these white space changes were in a commit on their own or were removed from this PR all together.